wherehows-web/app/components/datasets/containers/health-score-gauge.ts

```diff
@@ -0,0 +1,39 @@
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+import { IDatasetHealth } from 'wherehows-web/typings/api/datasets/health';
+import { readDatasetHealthByUrn } from 'wherehows-web/utils/api/datasets/health';
+import { get, set } from '@ember/object';
+
+// import { once } from '@ember/runloop';
+
+export default class HealthScoreGauge extends Component {
+  /**
+   * The urn identifier for the dataset
+   * @type {string}
+   */
+  urn: string;
+
+  /**
+   * Defines the tag to be used in the rendered html element for this component
+   * @type {string}
+   */
+  tagName: '';
```

Comment:
`tagName: '';`? you're declaring tagName can only be a literal of `''`. Don't think that's what you intend?